cvat/apps/engine/frame_provider.py

```diff
@@ -175,7 +174,7 @@ def get_preview(self, frame_number):
         else:
             preview, _ = self.get_frame(frame_number, self.Quality.COMPRESSED, self.Type.PIL)
 
-        preview = rotate_within_exif(preview)
+        preview = ImageOps.exif_transpose(preview)
```

Comment:
```suggestion
        ImageOps.exif_transpose(preview, in_place=True)
```

Reply:
in_place is not supported in Pillow 9.3.0.
It was implemented starting Pillow 10